migration/1678950910841-insert_polygon_givback_eligible_tokens.ts

```diff
@@ -0,0 +1,306 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+import { Token } from '../src/entities/token';
+import { NETWORK_IDS } from '../src/provider';
+
+const polygonTokens = [
```

Comment:
@divine-comedian  I would appreciate if you take a look on tokens and addresses, and about `mainnetAddress` we need it for:

>Some tokens like PAN, XNODE, CRV dont have price on coingecko for gnosis network, So frontend guys suggested
add  mainnetAddress field for those tokens, then client can get price of these tokens in mainnet